.github/workflows/lint.yml

```diff
@@ -0,0 +1,30 @@
+# Lint tests run on PR
+# but should not run after push to master because reporting
+# these after push is meaningless to the building of the package
+name: python lint
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
+        python-version: [3.6]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2.2.2
```

Comment:
try `uses: actions/setup-python@v2` in some other projects I had `2.2.2` and some errors were happening. Keep in mind this for the future, if you don't change now, and something breaks, this might be a suspect.